SocketRocket/SRWebSocket.m

```diff
@@ -1063,7 +1063,10 @@ - (void)_pumpWriting;
             return (sentLength >= (NSInteger)size); // If we can't write all the data into the stream - bail-out early.
         });
         if (streamFailed) {
-            NSError *error = SRErrorWithCodeDescriptionUnderlyingError(2145, @"Error writing to stream.", _outputStream.streamError);
+            NSInteger code = 2145;
+            NSString *description = @"Error writing to stream.";
+            NSError *streamError = _outputStream.streamError;
+            NSError *error = _outputStream.streamError ? SRErrorWithCodeDescriptionUnderlyingError(code, description, streamError) : SRErrorWithCodeDescription(code, description);
```

Comment:
Nit: Reuse `streamError` instead of getting the value for a property once again.